### ISSUE TITLE
Build failure with GHC 9.0

### ISSUE THREAD

**Issue by Lysxia** [kcsongor/generic-lens#132]
The Hackage releases now needs an upper bound on base. Error from building generic-lens-core-2.1.0.0 with GHC 9:

```
src/Data/Generics/Internal/Profunctor/Iso.hs:76:7: error:
    • Couldn't match type: p0 i0 a b -> p0 i0 s t
                     with: forall (p :: * -> * -> * -> *) i.
                           Profunctor p =>
                           p i a b -> p i s t
      Expected: (s -> a) -> (b -> t) -> Iso s t a b
        Actual: (s -> a) -> (b -> t) -> p0 i0 a b -> p0 i0 s t
    • In the expression: dimap
      In an equation for ‘iso’: iso = dimap
    • Relevant bindings include
        iso :: (s -> a) -> (b -> t) -> Iso s t a b
          (bound at src/Data/Generics/Internal/Profunctor/Iso.hs:76:1)
   |
76 | iso = dimap
   |       ^^^^^
```

**Comment by anka-213** [comment]
I believe this was resolved by #130. The only thing that's missing is for it to be released to hackage.

**Comment by newhoggy** [comment]
Please do

**Comment by blamario** [comment]
Yes, relase it please.

**Comment by gwils** [comment]
Please release

**Comment by amesgen** [comment]
If you are blocked by this in an application, you can use the [`soure-repository-package`](https://cabal.readthedocs.io/en/3.4/cabal-project.html#specifying-packages-from-remote-version-control-locations):

```cabal
source-repository-package
    type: git
    location: https://github.com/kcsongor/generic-lens
    tag: 8e1fc7dcf444332c474fca17110d4bc554db08c8
    subdir: generic-lens-core generic-lens generic-optics
```

Stack also has a similar feature I think.

**Comment by kcsongor** [comment]
Making a release now

**Comment by kcsongor** [comment]
#130 is now released on 2.2.0.0
